Core/clim-core/table-formatting.lisp

```diff
@@ -66,38 +82,57 @@
   (:default-initargs
    :align-x :left :align-y :baseline :min-width 0 :min-height 0))
 
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (alexandria:define-constant +standard-cell-output-record-initargs+
+      '(align-x align-y min-width min-height)
+    :test #'equal))
```

Comment:
these initargs are initargs of the protocol class `cell-output-record`, not the `standard-cell-output-record`, please rename the constant.

Reply:
I gather that these are not keywords in order to include them in defgeneric as arguments - please define initargs as keywords and use the constant without #.(map 'list …) later.

Mention arguments in functions and methods "manually".